docs/roles/secretary.rst

```diff
@@ -9,22 +9,33 @@ This page explains more about the responsibilities and role of the secretary.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
-
+The secretary is responsible for scheduling, advertising, and recording RITlug general and eboard meetings.
 
 Eligibility requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-< basic eligiblity requirements – what if someone is on co-op? away from campus? working remote? is under academic suspension (i.e. low GPA)? >
+* Must be a full time student on main campus for at least one semester
+* Cannot be on academic probation
+* Must be present for **75% of weekly club meetings and eboard meetings**
 
 
 Responsibilities
 ----------------
 
-< responsibilities and routine work that someone in this position is directly accountable for – what do they do and what should other eboard members expect from this person? >
+* *Weekly Presentations*
+   * Review of presentations
+   * Addition of presentations to RITlug website
+* Record meeting minutes during both general and eboard meetings
+* Setup live stream or video conference for meetings and presentations that require remote participants or viewers
+* Take attendance of both general and eboard meetings
+* Prepare the agenda for eboard or special meetings
+* Manage calendar events for meetings and events
+* Manage mailing list participants
```

Comment:
Could you give an example of this? Like if someone asks to be unsubscribed or if someone isn't receiving emails, should this mean we direct them to the secretary?

Reply:
I am a little torn between what this responsibility actually entails. On one hand, it's redundant with the "Campus groups member management" responsibility, but then there's the Google group for alumni announcements. I need some advisement here.

Or this could be reworded to have the secretary send announcements to the mailing list.

Reply:
I would personally just take this out of the list. It's not really a specific role of the secretary to keep track of anything with the mailing list.